src/App/Router.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Friendica\App;
+
+
+use FastRoute\DataGenerator\GroupCountBased;
+use FastRoute\Dispatcher;
+use FastRoute\RouteCollector;
+use FastRoute\RouteParser\Std;
+use Friendica\Module\Itemsource;
+
+class Router
+{
+	/** @var RouteCollector */
+	public $routeCollector;
+
+	public function __construct(RouteCollector $routeCollector = null)
+	{
+		if (!$routeCollector) {
+			$routeCollector = new RouteCollector(new Std(), new GroupCountBased());
+		}
+
+		$this->routeCollector = $routeCollector;
+	}
+
+	public function getModuleClass($cmd)

```

Comment:
PhpDoc

Reply:
What about adding a "basic" test? Not that much but as a boilerplate for further enhancements to the class

```php
public function testGetModuleClass()
{
$router = new Router();
$cmd = 'api?guide=test';
$class = $router->getModuleClass($cmd);

$this->assertEquals('api', $class);

}
```